src/patternfly/sass-utilities/colors.scss

```diff
@@ -2,20 +2,20 @@
 $pf-color-black-100:             #fafafa !default;
 $pf-color-black-150:             #f5f5f5 !default;
 $pf-color-black-200:             #ededed !default;
-$pf-color-black-300:             #d1d1d1 !default;
+$pf-color-black-300:             #d2d2d2 !default;
 $pf-color-black-400:             #bbb !default;
 $pf-color-black-500:             #8b8d8f !default;
 $pf-color-black-600:             #72767b !default;
 $pf-color-black-700:             #4d5258 !default;
 $pf-color-black-800:             #393f44 !default;
-$pf-color-black-900:             #282d33 !default;
+$pf-color-black-900:             #151515 !default;
 $pf-color-black-1000:            #030303 !default;
 $pf-color-blue-50:               #def3ff !default;
 $pf-color-blue-100:              #bee1f4 !default;
-$pf-color-blue-200:              #7dc3e8 !default;
-$pf-color-blue-300:              #39a5dc !default;
-$pf-color-blue-400:              #007bba !default;
-$pf-color-blue-500:              #00659c !default;
+$pf-color-blue-200:              #2b9af3 !default;
+$pf-color-blue-300:              #73bcf7 !default;
```

Comment:
Are 200 and 300 reverse? 200 looks darker than 300 to me.
<img width="407" alt="Screen Shot 2019-03-20 at 1 28 19 PM" src="https://user-images.githubusercontent.com/4032718/54705854-124d5d80-4b14-11e9-8f02-6013c3a2bd6d.png">

Reply:
Yes, good catch! @kybaker what do you think? This matches the updates from the spreadsheet.

Reply:
Makes sense to me as long as it doesn't change any of the components or the hex values in the spreadsheet.

Reply:
Thanks @kybaker @mindreeper2420 - updated.